src/Symfony/Component/Inflector/Tests/InflectorTest.php

```diff
@@ -78,12 +79,12 @@ public function singularizeProvider()
             ['formulae', 'formula'],
             ['formulas', 'formula'],
             ['fungi', 'fungus'],
-            ['funguses', ['fungus', 'funguse', 'fungusis']],
+            ['fungi', ['fungus', 'funguse', 'fungusis']],
             ['garages', ['garag', 'garage']],
             ['geese', 'goose'],
             ['halves', ['half', 'halve', 'halff']],
             ['hats', 'hat'],
-            ['heroes', ['hero', 'heroe']],
+            ['heroes', ['hero', 'hero']],
```

Comment:
having it twice looks weird to me

Reply:
Oops, this happened on accident while backporting my fix from 5.2 to 4.4 branch. Updated.

Reply:
Somehow this happens by fabpot.io 🤔  curl https://fabbot.io/patch/symfony/symfony/41105/8b2c0979505f1a43e6311111d48ebde8ba25de28/typos.diff | patch -p0

Reply:
Not sure how to solve this

Reply:
well, ignore it. It is totally expected to have typos in this test, as only one of the returned alternatives will be the valid word generally.